index.js

```diff
@@ -16,6 +18,7 @@ async function docker_buildx() {
         const buildArg = extractInput('buildArg', false, '');
         const target = extractInput('target', false, '');
         const context = extractInput('context', false, '.');
+        core.info(`Running ${publish ? 'buildAndPublish' : 'buildOnly'}`)
```

Comment:
Insert this kind of information inside the buildAndPublish and buildOnly functions please